public/index.html

```diff
@@ -31,11 +31,62 @@
     </style>
   </head>
   <body>
-    <div id="message">
-      <h2>Welcome to</h2>
-      <h1>Past Paper Hub Test</h1>
-      <h1>by Rich Kitibwa</h1>
+    <div id="container">
+      <div class="header">
+          <div class= "Subhead">
+            <h1> <strong> PastPaper Hub </strong></h1>
+            
+        </div>     
     </div>
+    
+    <div class="Menu"> 
+          <ul class="main">
+              <li> <a href="#"> University </a>
+                <ul class="Menulist"> 
+                    <li> Makerere  </li>
+                    <li> Gulu </li>
+                    <li> MUST </li>
+                </ul>
+            
+             </li>
+              <li> <a href="#">Year</a>
+                <ul class="Menulist"> 
+                        <li>One</li>
+                        <li>Two</li>
+                        <li>Three</li>
+                    </ul>
+            
+             </li>
+              <li> <a href="#">Course </a>
+                <ul class="Menulist"> 
+                        <li>Medicine</li>
+                        <li>Engineering</li>
+                        <li> Economics</li>
+                    </ul>
+            
+             </li>
+              <li><a href="#"> Course Unit </a>
+                <ul class="Menulist"> 
+                        <li>Biochemistry</li>
+                        <li>Machine language</li>
+                        <li> Bsiness computing</li>
+                    </ul>
+            
+             </li>
+          </ul>
+      </div>
+      
+      <div class="Mainbody">
+            <p> Preparing for exams and tests is one of the hard times most students go through while in school. PastPaper Hub was created bearing in mind the frustration of such  students ; giving  access to past papers and answers and also creating interactive discussions to enable students discuss some of the solutions.  </p>
```

Comment:
realign  these so that they are more organised. I can do this for you if you like?

Reply:
Also this line exceeds 120 characters. A line of code should not be that long. anything beyond 120 characters should go to the next line. check https://stackoverflow.com/a/29972073/2534091 on how to add a line to your Visual Studio IDE

Reply:
Ok i will implement all the changes suggested